wallaby.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(wallaby) {
```

Comment:
should this file be renamed to wallaby.conf.js? following the config filename structure we see at karma?